metrics/bucket.go

```diff
@@ -0,0 +1,51 @@
+package metrics
+
+import (
+	"context"
+
+	"go.opencensus.io/stats"
+	"go.opencensus.io/stats/view"
+	"go.opencensus.io/tag"
+)
+
+// NewInt64ByteBucket creates a Int64Bucket with units of bytes and a default set of aggregation
+// bounds from .25 kilobytes to 10 megabytes.
+func NewInt64ByteBucket(name, desc string, tagKeys ...tag.Key) *Int64Bucket {
+	defaultBounds := []float64{250, 500, 1000, 5000, 10000, 50000, 100000, 500000, 1000000, 5000000, 10000000}
+	return NewInt64BucketWithBounds(name, desc, stats.UnitBytes, defaultBounds, tagKeys...)
+}
+
+// NewInt64BucketWithBounds creates a Int64Bucket wrapping an opencensus int64 measurement.
+func NewInt64BucketWithBounds(name, desc, unit string, bounds []float64, tagKeys ...tag.Key) *Int64Bucket {
+	log.Infof("registering avg counter: %s - %s", name, desc)
```

Comment:
Wrong comment. Maybe it's possible to use reflection to produce a comment that can be safely copy/pasted?